### DISCUSSION TITLE
'underscored' file names are not getting skipped

### DISCUSSION THREAD

**Discussion by ayushsharma82** [expo/router#395] [Ideas]
### Summary

I expected expo router to work almost in the same convention as Nextjs router, hence I tried 'underscoring' the file names to skip them from navigation, but it didn't work and I was rather navigated to `_index.jsx` file.

I feel this feature should be implemented where expo router smartly skips any file which has an underscore added in front of it. This is quite useful when you have to selectively render a page under the same route.

Other than this small issue, Expo Router is very promising!

### Minimal reproducible example

--

**Comment by marklawlor** [comment]
Converting this to a discussion, as Expo Router does not currently offer this functionality and it should be discuessed further. Co-location is something the project will address, but we first need to ensure that base functionality is implemented (see v2 roadmap https://github.com/expo/router/discussions/309)

Traditionally underscores have been a convention, but have recently fallen out of favour. It would be interesting to hear the community thoughts if they are for/against them.
